packages/jest-expect-har/package.json

```
@@ -33,7 +42,8 @@
     "url": "https://github.com/readmeio/oas/issues"
   },
   "scripts": {
-    "build": "tsc",
+    "attw": "attw --pack --format ascii --profile node16",
```

would it pass without this flag? might be a good check to have to ensure this is a non-breaking change

```suggestion
    "attw": "attw --pack --format ascii",
```

---

yep `node10` still works, will leave `--profile` off